packages/client/src/getChainId.ts

```diff
@@ -0,0 +1,33 @@
+import { ZetaChainClient } from "./client";
+
+/**
+ * Get chain ID from a chain label.
+ *
+ * @param this - ZetaChainClient instance.
+ * @param chainNameOrAlias Chain label like goerli_testnet
+ * @returns
+ */
+export const getChainId = function (
+  this: ZetaChainClient,
+  chainNameOrAlias: string
+): number | null {
+  const networks = this.chains;
+  if (networks.hasOwnProperty(chainNameOrAlias)) {
+    return networks[chainNameOrAlias as keyof typeof networks].chain_id;
+  }
+
+  // Iterate through networks to check aliases
+  for (const key in networks) {
+    if (networks.hasOwnProperty(key)) {
+      const chain: any = networks[key as keyof typeof networks];
```

Comment:
Can't do it right now, because the exported type in the networks package does not match exactly what's in the networks JSON (specifically, it doesn't have chain aliases), created an issue for this: https://github.com/zeta-chain/networks/issues/44